custom/kodi/areascout/gbm/overlay/etc/systemd/system/kodi.service

```diff
@@ -6,8 +6,9 @@ StartLimitIntervalSec=0
 Type=simple
 Restart=no
 RestartSec=1
+User=odroid
```

Comment:
'User=@@DEFAULT_USER@@' would be nice and @@DEFAULT_USER@@ will be replaced to default user account when building an image.

Reply:
Changed :)